client/src/i18n/locales/en.json

```diff
@@ -38,6 +38,8 @@
     "AboutMistakesJournal": {
       "HeaderFirstPart": "Get the most out of your mistakes",
       "HeaderSecondPart": "by keeping a",
+      "EarlyAccessHeader": "Welcome to Early Access!",
+      "EarlyAccessText": "See for yourself and tell us what you think!",
```

Comment:
Too many `!` for me. I would remove `!` from `EarlyAccessText` and just ad regular `.`

Reply:
Szymons translation